### PR TITLE
{bio}[foss/2018a] MaSuRCA v3.2.4

### PR DESCRIPTION
**Pull request by ebirn**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.4-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+# need a temporary url to download it. Do it manually here:
+# http://www.genome.umd.edu/masurca_form.html
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['759d5b0411b048d996df1ca6daadf1cc49ff88f4436a21cd81d7f191a8bd80b0']
+
+dependencies = [
+    ('libreadline', '6.3', '', True),
```
Comment (boegel) [inline easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.4-foss-2018a.eb]: @ebirn This should be `7.0`, and you shouldn't force the `dummy`-built `libreadline` here?

Comment (ebirn) [reply easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.4-foss-2018a.eb]: I guess it should, didn't see the v7.0 when I put this together. Generally though: if there is a 'foss' and a 'GCCcore' version of a dependency, the 'GCCcore' would take precedence?

Comment (boegel) [reply easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.4-foss-2018a.eb]: By default, no, the `foss` version would be picked up by default, only when that is missing `eb` will consider the `GCCcore` one.

Unless you have `--minimal-toolchains` enabled in your EasyBuild configuration, then it's the reverse (cfr. http://easybuild.readthedocs.io/en/latest/Manipulating_dependencies.html#using-minimal-toolchains-for-dependencies).

File: easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.4-foss-2018a.eb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+description = '''MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph  
+ and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing 
+ only short reads from Illumina sequencing or a mixture of short reads and long reads 
+ (Sanger, 454, Pacbio and Nanopore).'''
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+# need a temporary url to download it. Do it manually here:
+# http://www.genome.umd.edu/masurca_form.html
```
Comment (boegel) [inline easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.4-foss-2018a.eb]: Can't seem to download the sources right now... `An error occurred when sending the email. `

Comment (ebirn) [reply easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.4-foss-2018a.eb]: True, I get the same thing. Eventually you end up there: ftp://ftp.genome.umd.edu/pub/MaSuRCA/latest
and it seems they might be moving to Github, but I didn't want to rely on this: https://github.com/alekseyzimin/masurca/releases (as the Github account seems to be a personal one, the original website is institutional). 
Should I change it for the direct FTP link?

Comment (boegel) [reply easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.4-foss-2018a.eb]: I'd say yes, if we can auto-download, we should?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2672.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4b573a98c77d52bc76f77cf8ce594f67 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2025.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5d9c83b8c1d222f92696cc569de62e8e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ebirn!

**Pull request merged**